src/components/Modal/Modal.tsx

```diff
@@ -22,14 +20,14 @@ function Modal(props: TModalProps) {
 
   return createPortal(
     <AnimatePresence>
-      {props.isOpen && (
+      {props.isOpen ? (
```

Comment:
Não usar ternário, usar o `&&`

Fale com o @alvarogfn sobre remoção da rule

Reply:
Não sei se concordo com isso não.

Pra mim ternário >>> &&.
https://dev.to/maafaishal/avoid-operator-for-conditional-rendering-in-react-2de